README.md

```diff
@@ -275,7 +276,7 @@ If your build machine also has some QT5 components, it is useful to `export QT_S
 ## How to run your Build
 
 In order to setup an SD card with a newly built version of NOOBS, you will need to:
-- Format an SD card that is 8GB or greater in size as FAT32
+- Format an SD card that is 16GB or greater in size as FAT32
```

Comment:
size -> capacity

Reply:
For the audience that NOOBS is aimed at, I think "size" might be more understandable than "capacity" ?

Reply:
Good point. Probably best to leave it at size. I'm trying to dance on the head of a pin again really.

Reply:
Just noticed that https://www.raspberrypi.org/help/noobs-setup/2/ uses "capacity" 😮